### ISSUE TITLE
Admins can't edit guest users who don't have email addresses

### ISSUE THREAD

**Issue by rmunn** [sillsdev/languageforge-lexbox#657]
Once #621 is merged, admins will eventually want to edit guest users who don't have email addresses. Currently trying to do so results in an error because the email is required:

![image](https://github.com/sillsdev/languageforge-lexbox/assets/90762/8e3dd87b-9124-4bf9-93ed-77b555226013)

We'll need to update the edit-user GQL query and backend code to deal with editing users will null emails. Some of that work has already been done, but enough remains that it's worth tracking this as a separate issue.

**Comment by rmunn** [comment]
Since @myieye has created a PR for this that seems to be complete, I'll assign this to him instead of me.
